packages/xo-server-audit/config.toml

```diff
@@ -0,0 +1,4 @@
+blockList = [
```

Comment:
Why `blockList`?

Reply:
I thought the comment was about the chose name.  

`blockList` is a list of methods which will not be recorded.

Reply:
Ah, I remember now! And yes, it was about the name because I really read it as "a list of blocks" instead of "a list of blocked methods". So maybe `blockedList` or `blackList` and/or a comment?

Reply:
I'm OK with `blockedList` with comment :+1: